### ISSUE TITLE
Editor sometimes fails to fully appear or show preview

### ISSUE THREAD

**Issue by mtmorgan** [Bioconductor/legacy.support.bioconductor.org#63]
the 'new' markdown-based editor does not show buttons above the editing box, or does not show a preview, or the preview update is very slow and interferes with text entry. This has come up on specific installations, e.g., https://github.com/Bioconductor/support.bioconductor.org/issues/62#issuecomment-456486279 but it is not clear what the underlying issue is.

A KNOWN issue is that the preview does not display when one posts a 'comment'; the issue under discussion here applies to new questions and answers.

**Comment by gksmyth** [comment]
I can tell you that the buttons and preview don't appear when I use Google Chrome under Windows. I am using the latest version of Chrome and I am not aware of anything unusual about my setup. I see the same behaviour on two different PCs, one running Windows 10 and other other Windows 7.

The buttons and preview do appear when I use Firefox under Windows.

I don't know what other hints to give. I am happy to answer questions if you have any.

**Comment by lshep** [comment]
@gksmyth  could you also try clearing your cache browser and browsing history?  We did have some quirks originally that cleared up with clearing those.

**Comment by gksmyth** [comment]
@lshep Yes, that's a good suggestion, and it worked. After clearing the cache/history/cookies, the edit buttons do indeed now appear for me. Perhaps I should have thought of trying that myself.

Previewing now appears for questions and answers but not comments, which is apparently the same as what others see.

I will have to wait a bit longer to confirm whether slow updating of the preview is also fixed.

**Comment by mtmorgan** [comment]
I wonder if other users also experience this, and that is the reason why they submit poorly formatted posts? The solution would be to force the javascript library to be reloaded, maybe using https://stackoverflow.com/a/32427/547331 ? (I recall vaguely this being used elsewhere in the code....)

**Comment by gksmyth** [comment]
@mtmorgan I can confirm that other people at the WEHI are having the same problem I did. In fact it's worse for them than it was for me because I at least knew (from Lori's email on Dec 20) that Bioconductor Support was converting to markdown, whereas other people just assume that the website has bugs.

**Comment by lshep** [comment]
I'll look into the forcing reload early next week.  Sorry for the inconvenience this is causing @gksmyth

**Comment by gksmyth** [comment]
@lshep Clearing the cache did solve the slow/jerky previewing for me temporarily, but it came back after a week. I had to clear my cache again today to get back to normal responsiveness.

**Comment by lshep** [comment]
Yes I'm working on adding in the fix for an automatic updated on the template pages so clearing the cache won't be necessary.  I'm still working out the details but we are trying to get this resolved.

**Comment by lshep** [comment]
I just committed a fix that I'm hoping will fix the issue.  Please keep me posted on if  you continue to have to clear the cache
